sandbox/technical_user_guide.md

```diff
@@ -136,17 +131,58 @@ Basic Mzg0MWM1OTQtYThjMC00MWU1LTk4Y2MtMzhiYjQ1MzYwZDNjOmY5NzgwZDMyMzU4OGYxY2RmYz
 * `Accept: application/json`
 * `Authorization: <Encoded Basic authentication>`
 
-**cURL command**
+**cURL commands**
+
+You can choose one of two cURL commands to use.
+
+**Option 1, which requires base-64 encoding be performed on your clientId and secret**
+
+Format:
+
+```
+curl -H "authorization: Basic [base64-encoded clientId:secret]"
+```
+
+In the following cURL command, we have concatenated the base64 encoding of the 'Client ID":" Client Secret' as the argument to the -H flag.
```

Comment:
Looks like there are some extraneous double-quotes in this sentence, around the colon.
Screenshot from local rendering:

<img width="660" alt="Screen Shot 2020-04-27 at 10 48 03 PM" src="https://user-images.githubusercontent.com/37818548/80441712-7591ed80-88d9-11ea-8f3c-83192a550dd7.png">